go/internal/feast/server/http_server.go

```diff
@@ -190,17 +191,19 @@ func (s *httpServer) getOnlineFeatures(w http.ResponseWriter, r *http.Request) {
 	for _, vector := range featureVectors {
 		featureNames = append(featureNames, vector.Name)
 		result := make(map[string]interface{})
-		var statuses []string
-		for _, status := range vector.Statuses {
-			statuses = append(statuses, status.String())
-		}
-		var timestamps []string
-		for _, timestamp := range vector.Timestamps {
-			timestamps = append(timestamps, timestamp.AsTime().Format(time.RFC3339))
-		}
+		if request.Status {
```

Comment:
Can you correct formatting?

Reply:
I am not sure why github is displaying this formatting. On my local it displays with the correct formatting and when I copy the code from github to my local IDE it shows the correct formatting again.